README.md

```diff
@@ -4,16 +4,38 @@ Node.js NAN C++ NuoDB Driver
 
 The node-nuodb add-on for Node.js powers high performance NuoDB Database applications.
 
-Use node-nuodb to connect Node.js 8 and 10 to NuoDB Database.
+Use node-nuodb to connect Node.js to a NuoDB Database.
 
 node-nuodb supports basic features of the NuoDB Database and NuoDB C++ client.
 
+## Dependencies
+
+Node 16.16.0+
+
 ## Installation
 
-The driver is distributed as Dockers; two variants exist, an ONBUILD
-variant, and a traditional variant. See the documentation for more
```

Comment:
Was it your intention to remove any information regarding docker?  I think we need to talk to Ariff about that.

Reply:
I wanted to remove any incorrect installation instruction, and as far as I am aware the driver is not distributed as a "Dockers".  I don't see any link between this driver and Docker anymore.